src/vs/workbench/contrib/accessibility/browser/accessibleView.ts

```diff
@@ -126,7 +126,7 @@ class AccessibleView extends Disposable {
 			this._register(this._editorWidget.onKeyUp((e) => {
 				if (e.keyCode === KeyCode.Escape) {
 					this._contextViewService.hideContextView();
-				} else if (e.keyCode === KeyCode.KeyD) {
+				} else if (e.keyCode === KeyCode.KeyD && this._configurationService.getValue(settingKey)) {
```

Comment:
Does reading a configuration key on every keyup even have a performance overhead?

Reply:
only when the key that's up is `D`

Reply:
oh, nevermind, this is only the accessible view

Reply:
which will be rarely in a readonly editor 😄